accounts/abi/bind/v2/internal/contracts/db/bindings.go

```diff
@@ -107,20 +123,19 @@ type GetNamedStatParamsOutput struct {
 // from invoking the contract method with ID 0xe369ba3b.
 //
 // Solidity: function getNamedStatParams() view returns(uint256 gets, uint256 inserts, uint256 mods)
-func (dB *DB) UnpackGetNamedStatParams(data []byte) (GetNamedStatParamsOutput, error) {
+func (dB *DB) UnpackGetNamedStatParams(data []byte) (*GetNamedStatParamsOutput, error) {
 	out, err := dB.abi.Unpack("getNamedStatParams", data)
 	outstruct := new(GetNamedStatParamsOutput)
 	if err != nil {
-		return *outstruct, err
```

Comment:
I don't think this is correct, you're ommitting the error here and the outstruct is != nil

Reply:
Oh, it's a mistake,  and it's fixed in this [commitment](https://github.com/ethereum/go-ethereum/pull/31692/commits/77ef9beb649405814a9d85bd20bcdedf881655dd).